packages/config/src/Config.ts

```diff
@@ -38,15 +35,15 @@ const DEFAULT_PREFER_RELATED_APPLICATIONS = true;
 export async function addPlatform(
   projectRoot: string,
   platform: 'ios' | 'android' | 'web'
-): Promise<{ exp: ?Object, pkg: ?Object, rootConfig: ?Object }> {
-  const { exp } = await readConfigJsonAsync(projectRoot);
+): Promise<{ exp: any; pkg: any; rootConfig: any }> {
```

Comment:
Instead of "any", how about "object" or perhaps an aliased type (`type ExpoConfig = object;`)?